File: HOL/dotnet/completed/DevCamp.WebApp/Utils/IncidentApiHelper.cs
using IncidentAPI;
using System;
using System.Configuration;

namespace DevCamp.WebApp.Utils
{
    public class IncidentApiHelper
    {
       public static IncidentAPIClient GetIncidentAPIClient()
        {
            var client = new IncidentAPIClient(new Uri(Settings.INCIDENT_API_URL));
            return client;
        }

    }
}

File: HOL/java/03-azuread-office365/start/src/main/java/devCamp/WebApp/Utils/IncidentAPIHelper.java
package devCamp.WebApp.Utils;

import devCamp.WebApp.IncidentAPIClient.IncidentAPIClient;

public class IncidentAPIHelper {
    public static IncidentAPIClient getIncidentAPIClient() {

        String apiurl= System.getenv("INCIDENT_API_URL");
        return new IncidentAPIClient(apiurl);
    }
}